wideq/client.py

```diff
@@ -97,6 +102,24 @@ def __init__(self,
         self._country: str = country
         self._language: str = language
 
+    def _inject_thinq2_device(self):
+        """This is used only for debug"""
+        data_file = (
+            os.path.dirname(os.path.realpath(__file__)) + "/deviceV2.txt"
+        )
+        with open(data_file, "r") as f:
+            device_v2 = json.load(f)
+        for d in device_v2:
+            self._devices.append(d)
+            LOGGER.debug("Injected debug device: %s", d)
+
+    def _load_devices(self, force_update: bool = False):
+        if self._session and (self._devices is None or force_update):
```

Comment:
This doesn't work as-is for two reasons:
- `self._session` is itself initialized lazily, so if that hasn't happened as a side effect of something else, we bail out. I suspect you really just want `self.session` instead to force the initialization to happen.
- self._devices is initialized to `[]`, which isn't `None`. I suspect you probably want to check `not self._devices` instead?